root/static/scripts/release-editor/actions.js

```diff
@@ -69,6 +75,83 @@ const actions = {
     }));
   },
 
+  duplicateReleaseGroups: ko.observableArray([]),
+  loadingDuplicateReleaseGroups: ko.observable(false),
+  failedLoadingDuplicateReleaseGroups: ko.observable(false),
+  duplicateReleaseGroupsRequest: null,
```

Comment:
I tried to make this property observable and get rid of `loadingDuplicateReleaseGroups`, but I noticed strange behavior where modifying the observable `duplicateReleaseGroupsRequest` from within `loadDuplicateReleaseGroups` seemed to make my `debounceComputed(utils.withRelease(...))` get called again immediately.